vcd/resource_vcd_solution_add_on_instance.go

```diff
@@ -0,0 +1,222 @@
+package vcd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceVcdSolutionAddonInstance() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceVcdSolutionAddonInstanceCreate,
+		ReadContext:   resourceVcdSolutionAddonInstanceRead,
+		UpdateContext: resourceVcdSolutionAddonInstanceUpdate,
+		DeleteContext: resourceVcdSolutionAddonInstanceDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceVcdSolutionAddonInstanceImport,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"add_on_id": {
+				Type:        schema.TypeString,
+				Required:    true,
```

Comment:
Should this one be also `ForceNew: true`? (that's what I understood from Update comments)

Reply:
Yes, totally - good point